### DISCUSSION TITLE
영상에 아이디 첫글자

### DISCUSSION THREAD

**Discussion by act-lamos** [metamong-friends/metamong-friends.github.io#12] [제안하기]
저희팀은 보통 영상을 안켜놓고 지내는데요 John과 Jay모두 J로 나오고 실제 아이디는 아래 작게 표시되어있어서 누가 누구인지 약간 헷갈리는 느낌이네요

**Comment by shlee303** [comment]
닉네임이 좀 더 잘 보이도록 UI 개선 검토하겠습니다. 의견 감사합니다!

**Comment by shlee303** [comment]
@act-lamos 카메라 off 상태에서 글자 1개->3개로 늘리는 조치 완료했습니다 :) 항상 좋은 의견 많이 주셔서 감사합니다~
